database-commons/src/main/java/io/cdap/plugin/util/DBUtils.java

```diff
@@ -104,148 +98,6 @@ public static DriverCleanup ensureJDBCDriverIsAvailable(Class<? extends Driver>
     }
   }
 
-  /**
-   * Given the result set, get the metadata of the result set and return
-   * list of {@link io.cdap.cdap.api.data.schema.Schema.Field},
-   * where name of the field is same as column name and type of the field is obtained using
-   * {@link DBUtils#getSchema(int, int, int)}
-   *
-   * @param resultSet result set of executed query
-   * @param schemaStr schema string to override resultant schema
-   * @return list of schema fields
-   * @throws SQLException
-   */
-  public static List<Schema.Field> getSchemaFields(ResultSet resultSet, @Nullable String schemaStr)
```

Comment:
these are being removed because the logic has been moved into SchemaReader?

Reply:
I removed this method addressing https://github.com/data-integrations/database-plugins/pull/46#discussion_r305006496 because i did not find usages of it.